libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/Fields.hs

```diff
@@ -109,6 +116,20 @@ data TxField era
   | WitnessesI [WitnessesField era] -- Inlines Witnesess Fields
   | AuxData (StrictMaybe (TxAuxData era))
   | Valid IsValid
+  deriving (Generic)
+
+instance
+  ( Era era
+  , ToExpr (TxBody era)
+  , ToExpr (TxWits era)
+  , ToExpr (Script era)
+  , ToExpr (Redeemers era)
+  , ToExpr (TxAuxData era)
+  , ToExpr (TxOut era)
+  , ToExpr (TxCert era)
+  , ToExpr (PParamsHKD StrictMaybe era)
+  ) =>
+  ToExpr (TxField era) -- TODO: consider relocating
```

Comment:
All of this `Test.Cardano.Ledger.Generic` will be up for removal next. So, we don't really need to worry about finding a perfect place for these types and instances.
```suggestion
  ToExpr (TxField era)
```